srcpkgs/python-PyAudio/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'python-PyAudio'
+pkgname=python-PyAudio
+version=0.2.11
+revision=1
+noarch=yes
+wrksrc="PyAudio-${version}"
+build_style=python-module
+pycompile_module="PyAudio.py"
+hostmakedepends="python-setuptools python3-setuptools"
+makedepends="python-devel python3-devel portaudio-devel"
+depends="python"
+short_desc="Python2 Bindings for PortAudio, a cross-platform audio IO library"
+maintainer="Cameron Nemo <camerontnorman@gmail.com>"
+license="MIT"
```

Comment:
https://travis-ci.org/void-linux/void-packages/jobs/428490379#L402
````
$ common/travis/xlint.sh
srcpkgs/python-PyAudio/template:14: license 'MIT', but no use of vlicense
````

Reply:
was worried that would pop up. couldn't find a license or copying file, it is just listed in the source header and on PyPi.

Reply:
I included the LICENSE file in the files directory.

Reply:
The license is present in the README:
````
======================================================================
PyAudio v0.2.11: Python Bindings for PortAudio.
======================================================================

See: http://people.csail.mit.edu/hubert/pyaudio/

PyAudio provides Python bindings for PortAudio v19, the cross-platform
audio I/O library. Using PyAudio, you can easily use Python to play
and record audio on a variety of platforms.

See INSTALL for compilation hints.

======================================================================

PyAudio : Python Bindings for PortAudio.

Copyright (c) 2006 Hubert Pham

Permission is hereby granted, free of charge, to any person obtaining
a copy of this software and associated documentation files (the
"Software"), to deal in the Software without restriction, including
without limitation the rights to use, copy, modify, merge, publish,
distribute, sublicense, and/or sell copies of the Software, and to
permit persons to whom the Software is furnished to do so, subject to
the following conditions:

The above copyright notice and this permission notice shall be
included in all copies or substantial portions of the Software.

THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.

======================================================================
````

Reply:
no need to vendor the license just

```
post_install() {
    sed -n '${linenostart},${linenoend}p < README > LICENSE
    vlicense LICENSE
}
```

Reply:
@maxice8 yep, that's exactly what I meant.